package.json

```diff
@@ -5,7 +5,7 @@
   "engine-strict": true,
   "engines": {
     "node": "16.16.0",
-    "yarn": "^1.22.19"
+    "yarn": "^1.22.10"
```

Comment:
This looks to have been accidentally downgraded

Reply:
ouch, .. was reverting nodemon, thanks for spotting this!